packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -275,8 +279,13 @@ abstract class FlameChartState<T extends FlameChart,
     super.dispose();
   }
 
+  @override
+  bool wantKeepAlive = true;
```

Comment:
Do you want to add comment that explains why we want to keep it alive?

Reply:
added `KeepAliveWrapper` so that this can be used for any view, and added dart doc to that class.